pywhy_graphs/functional/linear.py

```diff
@@ -0,0 +1,104 @@
+from typing import Callable, List, Optional
+
+import networkx as nx
+import numpy as np
+
+
+def make_graph_linear_gaussian(
```

Comment:
suggesting doing this than all of that checking for None below

Reply:
Note that one should avoid lists etc. as default values, since they could be modified in the code and this would also change the default values (i.e., in Python, the defaults are pointing to the same mutable object). I guess the approach via `None` is the best here and then just clarify these defaults in the docstring.